python/yugabyte_db_thirdparty/builder.py

```diff
@@ -406,7 +419,7 @@ def add_include_path(self, include_path: str) -> None:
             log("Adding an include path: %s", include_path)
         cmd_line_arg = f'-I{include_path}'
         self.preprocessor_flags.append(cmd_line_arg)
-        self.compiler_flags.append(cmd_line_arg)
+        # self.compiler_flags.append(cmd_line_arg)
```

Comment:
Should we be commenting these out or removing them entirely? For here and all the other occurrences.